Category/categoryRouter.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express')
+const router = express.Router();
+const Category = require('./category.js');
+
+const handleError = (err, req, res, next) => {
```

Comment:
We can move this `handleError` function into a separate file so it's re-usable in all of your code!